src/lib/database.js

```diff
@@ -0,0 +1,13 @@
+var Sequelize = require('sequelize');
+var sql;
+var sqlInited;
+
+module.exports = {
```

Comment:
Here we can simply export the `sql` connection object without a `initSql` function.